### PR TITLE
Misspelling fixes

### PR DESCRIPTION
**Pull request by rhcarvalho**
Fixes several typos, inspired by #9091.

### CODE REVIEW

File: test/extended/networking.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ DEFAULT_SKIP_LIST=(
   "\[Feature:Federation\]"
 
   # Skipped until https://github.com/openshift/origin/issues/11042 is resolved
-  "should preserve source pod IP for traffic thru service cluster IP"
+  "should preserve source pod IP for traffic through service cluster IP"
```
Comment (stevekuznetsov) [inline test/extended/networking.sh]: Incorrect

File: pkg/generate/dockercompose/lookup.go [hunk 2]
```diff
@@ -12,11 +12,10 @@ import (
...
 )
 
-// TemplateFileSearcher resolves template files into template objects
-type FileSearcher struct {
-}
+// FileSearcher resolves template files into template objects.
+type FileSearcher struct{}
 
-// Search attempts to read template files and transform it into template objects
+// Search reads template files and transform them into template objects.
```
Comment (stevekuznetsov) [inline pkg/generate/dockercompose/lookup.go]: `s/transform/transforms/`

File: test/extended/networking.sh (hunk 1 above)
Comment (rhcarvalho) [inline test/extended/networking.sh]: http://www.dictionary.com/browse/thru
```
preposition, adverb, adjective
1. an informal, simplified spelling of through.
```

Why is that incorrect?

File: pkg/generate/dockercompose/lookup.go (hunk 2 above)
Comment (rhcarvalho) [inline pkg/generate/dockercompose/lookup.go]: Updated, thanks.

**Comment by rhcarvalho** [comment]
> I would not suggest making this into an automated linting step

Nobody suggested that.

File: test/extended/networking.sh (hunk 1 above)
Comment (stevekuznetsov) [inline test/extended/networking.sh]: These are literal names of tests to be skipped, and you did not change the name of the test. This is a correct misspelling but an incorrect change to the program.

Comment (liggitt) [inline test/extended/networking.sh]: https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/test/e2e/service.go#L232

**Comment by stevekuznetsov** [comment]
> Nobody suggested that.

 Sorry, I figured since you had bothered to script it and automate it you would also then want to add it as a linting step to run every so often.

**Comment by rhcarvalho** [comment]
@stevekuznetsov as you pointed out, scripting this would never work in practice ;)

The one-liner in the description is to make clear what was done (so that obvious mistakes can be spotted), and to avoid human mistakes from manually editing files (didn't want to introduce new typos :smile: ).

**Comment by stevekuznetsov** [comment]
If it only touched comments, it looks like it might be useful, actually. Would help keep the number of "fixed a typo in Godoc" PRs to 0, which I think most of us would appreciate since it will reduce the load on the merge queue and our review time budgets. Unfortunately when we use sets of strings instead of a type system in Go code or similar things in Bash unless it's not touching string literals it can't be automated.

**Comment by rhcarvalho** [comment]
> keep the number of "fixed a typo in Godoc" PRs to 0

:smile: while we are able to detect some things with automation, we would not be able to exterminate that class of PRs completely.

As an example, you pointed out a `s/transform/transforms` that I missed -- typos may be misspellings, wrong use of words, missing words, extra words, and then there are grammar errors, ...

**Comment by rhcarvalho** [comment]
bump

**Comment by stevekuznetsov** [comment]
LGTM [merge]

**Comment by stevekuznetsov** [comment]
Agh, forgot we're still in code freeze, which is why this didn't get the merge tag before, @rhcarvalho we'll just need to wait

**Comment by rhcarvalho** [comment]
@stevekuznetsov np, thanks.

**Comment by rhcarvalho** [comment]
@stevekuznetsov maybe now the merge queue is ready to get those typos fixed?

**Comment by stevekuznetsov** [comment]
@danmcp can you re-tag this one for merge? Conflicts broke the last job

**Comment by rhcarvalho** [comment]
Fixed conflict

**Comment by rhcarvalho** [comment]
Please re-tag for merge.

**Comment by stevekuznetsov** [comment]
c'mon Rosie [merge]

**Pull request merged**